samples/zephyr/hello1/mps2_an505.overlay

```diff
@@ -0,0 +1,9 @@
+/* DTS overlay file.
```

Comment:
Did this file get left around after the rebase? This directory no longer exists.